src/main/webapp/login-script.js

```diff
@@ -1,25 +1,35 @@
-function onSignIn(googleUser) {
+async function onSignIn(googleUser) {
```

Comment:
this file is also probably something to git rm or otherwise make sure is the same between us